docs/framework-instana_agent.md

```diff
@@ -0,0 +1,35 @@
+# Instana Agent Framework
+[IBM Instana](https://www.ibm.com/products/instana)  is the only real-time full-stack observability solution: zero sample tracing
+
+The Instana Agent Framework causes an application to be automatically configured to work with a bound Instana instance
+
+<table>
+  <tr>
+    <td><strong>Detection Criterion</strong></td><td>Existence of a single bound Instana service.
+      <ul>
+        <li>Existence of a Instana service is defined as the <a href="http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES"><code>VCAP_SERVICES</code></a> payload containing a service who's name, label or tag has <code>instana</code> as a substring with at least `agentkey` and `endpointurl` set as credentials.</li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Tags</strong></td>
+    <td><tt>instana-agent=&lt;version&gt;</tt></td>
```

Comment:
I guess the `version` only allows `latest` because that is what is supported for the download [here](https://github.com/cloudfoundry/java-buildpack/pull/1064/files#diff-0b65e1e21c904f165e0f41cda5aa5b6c4b3d1ee6337f8c9d54e00b2eaaddd1e6R76)?

Reply:
Tags are printed on stdout by buildpack script.
The stdout will always display the name of the agent and the version.
In our case since we don't publish the version yaml in the specified format publicly.
We will always use the _latest_ in the version. 
The agentkey and endpointurl are used in detection to inject instana agent.